src/main/java/teammates/common/util/Utils.java

```diff
@@ -35,5 +37,23 @@ public static Gson getTeammatesGson() {
                 .disableHtmlEscaping()
                 .create();
     }
+    
+    /**
+     * Convert a list of {@link FeedbackParticipantType} to a string where each element will be separated by
+     * a delimiter.
+     */
+    public static String joinParticipantTypes(List<FeedbackParticipantType> participants, String delimiter) {
```

Comment:
For the unit test, I can't see where unit test for the whole Utils are. Should I add a File under `test.cases.common`